support/entry/CMakeLists.txt

```diff
@@ -27,6 +27,10 @@ if (NOT OUTPUT_FILE)
     set(OUTPUT_FILE output.ppm)
 endif()
 
+if (NOT SHADER_COMPILER)
+    set(SHADER_COMPILER glslc-glsl)
+endif()
+
```

Comment:
You are going to want a line similar to 47 below.
SET(SHADER_COMPILER ${SHADER_COMPILER} CACHE STRING "description")

This will make sure it ends up in CMakeCache.txt. If it is missed from CMakeCache it will not persist between multiple rebuilds.